components/experimental/Cargo.toml

```diff
@@ -5,7 +5,7 @@
 [package]
 name = "icu_experimental"
 description = "ICU4X pre-release components under active development; all code in this crate is unstable."
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
why are we not using prerelease tags for `icu_experimental`? we're just burning one major version per beta

Reply:
Good call.

Reply:
Since they're prerelease versions, not postrelease versions, I made it 0.3.0-beta2, and we can make it 0.3.0 for ICU4X 2.0.